crmd/lrm.c

```diff
@@ -1821,9 +1865,33 @@ construct_op(lrm_state_t * lrm_state, xmlNode * rsc_op, const char *rsc_id, cons
     op->timeout = crm_parse_int(op_timeout, "0");
     op->start_delay = crm_parse_int(op_delay, "0");
 
+    rsc = lrm_state_get_rsc_info(lrm_state, rsc_id, 0);
```

Comment:
Let's move this (and the associated free) inside the "if" block, so we don't allocate memory when we don't need to.